packages/components/src/ListView/Header/Header.component.js

```diff
@@ -35,26 +38,30 @@ function getAction(action, index) {
 
 export function renderActions(headerDefault = []) {
 	if (headerDefault.length) {
-		return (
-			<div className="actions">
-				{headerDefault.map(getAction)}
-			</div>
-		);
+		return <div className="actions">{headerDefault.map(getAction)}</div>;
 	}
 	return null;
 }
 
-function Header({ headerDefault, headerLabel, nbItemsSelected, nbItems, required }) {
+function Header({ headerDefault, headerLabel, nbItemsSelected, nbItems, required, t }) {
 	function renderTitle() {
-		if (headerLabel) {
-			return <strong>{headerLabel}{required && '*'}</strong>;
-		}
-		return null;
+		const computedHeaderLabel = headerLabel || t('LISTVIEW_HEADER_TITLE', 'Values');
+		return (
+			<strong>
+				{computedHeaderLabel}
+				{required && '*'}
+			</strong>
+		);
 	}
 
 	function renderCount() {
 		if (nbItems >= 1 && nbItemsSelected >= 1) {
-			return <small>{`(${nbItemsSelected}/${nbItems} selected)`}</small>;
+			return (
+				<small>{`(${nbItemsSelected}/${nbItems} ${t(
+					'LISTVIEW_HEADER_SELECTED',
+					'selected',
```

Comment:
is this possible to inclllude  both nbItems as variable of the translation string ?